.github/workflows/centos7.yml

```diff
@@ -119,24 +114,27 @@ jobs:
            cd _build
            cpack -G TGZ
 
+  upload:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
     - name: Installer TGZ push
       uses: actions/upload-artifact@v4
       with:
         name: targz
-        path: _build/*.tar.gz
+        path: /build/*.tar.gz
 
     - name: Installer RPM push
       uses: actions/upload-artifact@v4
       with:
         name: rpm
-        path: _build/*.rpm
+        path: /build/*.rpm
 
     - name: Publish assets
       if: ${{ env.IS_RELEASE == 'true' }}
       env:
         GITHUB_TOKEN: ${{ github.token }}
         tag: ${{ github.event.inputs.release_tag }}
       run: |
-        gh release upload "$tag" _build/*.tar.gz _build/*.rpm
-
-  
\ No newline at end of file
+        apt install -y gh
```

Comment:
Installed by default on [GH's virtual environments](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#cli-tools), not needed